packages/react/src/components/HotspotEditorModal/__snapshots__/HotspotEditorModal.story.storyshot

```diff
@@ -730,7 +732,7 @@ exports[`Storybook Snapshot tests and console checks Storyshots Watson IoT Exper
               </div>
               <button
                 className="iot--dynamic-hotspot-source-picker__clear-button iot--dynamic-hotspot-source-picker__clear-button--invisible iot--btn bx--btn bx--btn--sm bx--btn--ghost bx--btn--icon-only bx--tooltip__trigger bx--tooltip--a11y bx--tooltip--top bx--tooltip--align-end"
-                data-testid="dynamic-hotspot-source-picker-clear-dropdown"
```

Comment:
should we add this real test id as a button?

Reply:
done!